pkg/defkinds/php/locks_test.go

```diff
@@ -45,7 +45,7 @@ func initSuccessfullyUpdateLocksTC(t *testing.T, mockCtrl *gomock.Controller) up
 		gomock.Any(),
 	).Return(rawDebianOSRelease, nil)
 
-	solver.EXPECT().FromBuildContext(gomock.Any()).Times(1)
+	// solver.EXPECT().FromBuildContext(gomock.Any()).Times(1)
```

Comment:
should this be reverted ?

Reply:
As this PR was already too big I decided to put tests fixes in my 2nd PR :wink: